src/main/res/xml/app_settings.xml

```diff
@@ -119,6 +119,13 @@
         android:title="@string/category_about">
 
         <Preference
+            android:title="@string/open_log"
+            android:summary="@string/open_log_summary">
+            <intent
+                android:action=".activities.LogActivity" />
```

Comment:
Okay I guess this is alright. I just don't completely understand the `DEFAULT` intent category.